src/index.tsx

```diff
@@ -1,4 +1,9 @@
 import { createRoot } from 'react-dom/client';
 import { App } from './App';
+import React, { StrictMode } from 'react';
```

Comment:
It's a common practice to import React at the beginning of the file, especially when using JSX syntax. Consider moving the React import to the top for better readability and convention adherence.